postcode_lookup/Makefile

```diff
@@ -0,0 +1,7 @@
+build-ECPostcodeLookupFunction:
+	pip install --break-system-packages setuptools
+	pip install --upgrade -r requirements.txt --target "$(ARTIFACTS_DIR)/python/"
+
+build-BasicAuthFunction:
+	pip install --break-system-packages setuptools
+	pip install --upgrade -r requirements.txt --target "$(ARTIFACTS_DIR)/python/"
```

Comment:
Hmm. Using `--break-system-packages` here feels wrong to me.

This maybe doesn't matter if this only ever runs in a CI environment or something but I definitely wouldn't want to run this on my local machine. It feels like what we should actually be doing here is running this in a clean ad-hoc virtualenv.

I think I'm going to tag in @symroe for thoughts on this one though as I'm not sure I have a concrete suggestion on where next.

Reply:
decision: leave this how it is for now